.github/workflows/ci.yml

```diff
@@ -85,3 +85,17 @@ jobs:
         with:
           severity: error
           ignore_paths: third-party
+  rspec:
+    name: ChefSpec
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@main
+      - uses: actionshub/chef-install@main
+        with:
+          version: 22.10.1013
```

Comment:
Is it required to specify a version? or can we just use the latest?

Reply:
Unfortunately no, because it would use Chef 18, while we enforce Chef 17 in our code.